packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -293,6 +310,8 @@ export class ColorPicker
     return this.color || this.previousColor || DEFAULT_COLOR;
   }
 
+  private isClearable: boolean;
```

Comment:
Can you move this so that it is in alphabetical order within its fellow private vars?